python/psautohint/autohint.py

```diff
@@ -29,23 +29,49 @@
 #  Save font plist file.
 
 import ast
+import copy
 import logging
 import os
 import re
 import time
 from collections import defaultdict, namedtuple
 
-from .otfFont import CFFFontData
+from .otfFont import (CFFFontData, convertT2GlyphToBez, convertBezToT2,
+                      interpolate_cff2_charstring)
+
```

Comment:
The current direction of this file is to keep it neutral to the font format, and encapsulate the glyph parsing and updating logic in the respective font classes. It would be really nice to keep it that way if it is not too much effort.

Reply:
Sure.